themes/src/main/resources/theme/base/admin/resources/js/controllers/clients.js

```diff
@@ -966,6 +966,12 @@ module.controller('ClientDetailCtrl', function($scope, realm, client, flows, $ro
         "request_uri only"
     ];
 
+    // KEYCLOAK-10747 Explicit Proof Key for Code Exchange Activation Settings
```

Comment:
I suggest to remove those lines with the JIRA reference from most of the places. IMO it is fine to keep in in the automated test, but I suggest to remove it from the other places in the codebase.

Reply:
I see, I'll get rid of JIRA reference comments from codebase other than arquillian integration test class.